src/Magnum/BulletIntegration/Test/CMakeLists.txt

```diff
@@ -24,5 +25,5 @@
 #   DEALINGS IN THE SOFTWARE.
 #
 
-corrade_add_test(BulletIntegrationTest IntegrationTest.cpp LIBRARIES MagnumBulletIntegration)
+corrade_add_test(BulletIntegrationTest IntegrationTest.cpp LIBRARIES MagnumBulletIntegration ${BULLET_LIBRARIES})
```

Comment:
This shouldn't be needed -- the bullet libraries are already a dependency of `MagnumBulletIntegration`.